.github/workflows/auto-patch-release.yml

```
@@ -1,17 +1,25 @@
 name: Automatically create patch release every 2 weeks
 on:
   schedule:
-    # Run every Friday right before midnight
-    - cron: '59 23 * * 5'
+    # Run every Thursday right before midnight
+    - cron: '59 23 * * 4'
+  workflow_dispatch:
 jobs:
   create_patch_release:
     name: Create release
     runs-on: ubuntu-latest
     steps:
+      - name: Determine if we skip cancel checks
+        id: skip-checks
+        shell: bash
+        # Skip checks if we are not a scheduled run
+        run: echo value=$(test ${{ github.event_name }} != schedule && echo true || echo false) >> "$GITHUB_OUTPUT"
+
       - name: Create release
         uses: dequelabs/axe-api-team-public/.github/actions/auto-patch-release-v1@main
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          skip_checks: ${{ steps.skip-checks.outputs.value }}
```

Could you not do the expression here rather than having a separate step to do it?

```suggestion
          skip_checks: ${{ github.event_name != schedule }}
```

---

We need the string true or false, so we use the output